packages/styles/src/themes/pentahoPlus.ts

```diff
@@ -390,41 +390,114 @@ const pentahoPlus = makeTheme((theme) => ({
     HvButton: {
       classes: {
         root: {
-          borderRadius: theme.radii.full,
+          border: "none",
+          borderRadius: 48,
           padding: theme.spacing(0, "sm"),
-          "&:active": {
-            boxShadow: `inset 0 1px 2px 0 #00000033`, // 33 = 20% opacity
+          [`&[data-color="positive"]`]: {
+            "&:hover": {
+              backgroundColor: theme.palette.green[700],
+            },
+            "&:active": {
+              backgroundColor: theme.palette.green[800],
+            },
           },
-        },
-        disabled: {
-          "&:active": {
-            boxShadow: "none",
+          [`&[data-color="negative"]`]: {
+            "&:hover": {
+              backgroundColor: theme.palette.red[700],
+            },
+            "&:active": {
+              backgroundColor: theme.palette.red[800],
+            },
```

Comment:
These (and most other) colors aren't taking the dark theme into consideration.

We could use something like [`light-dark()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/light-dark) for now, or wait for the new semantic tokens to be set

Either way, we shouldn't disregard the theming

Reply:
Agreed, we can now take advantage of the token added in [this PR](https://github.com/lumada-design/hv-uikit-react/pull/4327) for the theming 🙏

Reply:
I'll review this PR with the new tokens to check where we can use those.